ember-cli-flash/declarations/flash/object.d.ts

```diff
@@ -11,6 +11,5 @@ declare module 'ember-cli-flash/flash/object' {
     allowExit(): void;
     timerTask(): void;
     exitTimerTask(): void;
-    destroy(): void;
```

Comment:
This was an `EmberObject` method. Public API `destroyMessage` will destroy the message and should be used instead.